### PR TITLE
Abb 131 162

### PR DESCRIPTION
**Pull request by DenisNosnitsin1970**
New records Abb 131 162 and stubs.

### CODE REVIEW

**Review by CarstenHoffmannMarburg — APPROVED** [review]

File: ParisBNF/abb/BNFabb131.xml [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+                                  <date calendar="creation" when-custom="6158" when="0665">12 October 665</date>?). 
+                                  What the 2nd year of the priesthood of <persName ref="PRS6071Krestodo">ʾabuna Krǝstodolu</persName> and
+                                  the 33rd year of the reign of <persName ref="PRS4015Fasilada">King Fāsiladas</persName>
+                                  result in 1665 (CE). The colophon mentions the scribe <persName ref="PRS14482YekunoAmlak" role="scribe">Yǝkʷǝnno ʾAmlāk</persName>
+                                  and appears to tell that the exemplar of the manuscript (<foreign xml:lang="gez">ʾabʾu</foreign>) originates from <placeName ref="INS0329DE">Dabra Dāgā ʾƎsṭifānos</placeName>.
+                              It is not fully clear whether C. Conti Rossini considers the colophon as referring to the work or to the manuscript only (but the last 
+                              would contradict to his 19th-century dating of the manuscript). M. Chaîne tacitly corrects the colophon to <date> the year of grace 7158 = 1665 CE</date>
```
Comment (eu-genia) [inline ParisBNF/abb/BNFabb131.xml:55]: date when="1665"?

File: ParisBNF/et/BNFet435.xml [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+            </publicationStmt>
+            <sourceDesc>
+                <msDesc xml:id="ms">
+                    <msIdentifier>
+                        <repository ref="INS0303BNF"/>
+                        <collection>Manuscrits orientaux</collection>
+                        <collection>Fonds éthiopien</collection>
+                        <collection>Griaule</collection>
+                        <idno>BnF Éthiopien 435 (Griaule 127)</idno>
+                        <altIdentifier>
```
Comment (eu-genia) [inline ParisBNF/et/BNFet435.xml:29]: why facs for altIdentifier?

File: ParisBNF/et/BNFet435.xml [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+                            <idno facs="https://gallica.bnf.fr/ark:/12148/btv1b531192159">Griaule 435</idno>
+                        </altIdentifier>
+                    </msIdentifier>
+                    <msContents>                     
+                        
+                        <msItem xml:id="ms_i1">                        
+                            <title type="complete" >Protective prayers</title>
```
Comment (eu-genia) [inline ParisBNF/et/BNFet435.xml:36]: remove space `">`

File: ParisBNF/et/BNFet435.xml [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+                    <term key="ChristianLiterature"/>
+                    <term key="Magic"/>
+                    <term key="MoPe"/>
+                </keywords>
+            </textClass>
+            <langUsage><language ident="en">English</language></langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="DR" when="2017-06-09">Created entity</change>
+            <change who="DR" when="2023-10-09">Added relation to deleted BNFgriaule343</change>
```
Comment (eu-genia) [inline ParisBNF/et/BNFet435.xml:92]: DR or DN?

File: ParisBNF/et/BNFet348.xml [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+                    <term key="Miracle"/>
+                    <term key="Homily"/>
+                    <term key="Hagiography"/>
+                </keywords>
+            </textClass>
+            <langUsage><language ident="en">English</language></langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="DR" when="2017-06-09">Created entity</change>
+            <change who="DR" when="2023-10-09">Added relation to deleted BNFgriaule343</change>
```
Comment (eu-genia) [inline ParisBNF/et/BNFet348.xml:84]: DR or DN?

Comment (eu-genia) [reply ParisBNF/et/BNFet348.xml:84]: is this change element correct?

File: ParisBNF/et/BNFet435.xml (hunk 4 above)
Comment (eu-genia) [reply ParisBNF/et/BNFet435.xml:92]: Is this change element correct?

File: ParisBNF/et/BNFet435.xml [hunk 6]
```diff
@@ -0,0 +1,102 @@
...
+                <p/>
+            </abstract>
+            <textClass>
+                <keywords>
+                    <term key="ChristianLiterature"/>
+                    <term key="Magic"/>
+                    <term key="MoPe"/>
+                </keywords>
+            </textClass>
+            <langUsage><language ident="en">English</language></langUsage>
```
Comment (eu-genia) [inline ParisBNF/et/BNFet435.xml:88]: Geez

File: ParisBNF/et/BNFet348.xml [hunk 7]
```diff
@@ -0,0 +1,94 @@
...
+            </abstract>
+            <textClass>
+                <keywords>
+                    <term key="ChristianLiterature"/>
+                    <term key="Miracle"/>
+                    <term key="Homily"/>
+                    <term key="Hagiography"/>
+                </keywords>
+            </textClass>
+            <langUsage><language ident="en">English</language></langUsage>
```
Comment (eu-genia) [inline ParisBNF/et/BNFet348.xml:80]: Geez

File: ParisBNF/abb/BNFabb162.xml [hunk 8]
```diff
@@ -0,0 +1,204 @@
...
+            <textClass>
+                <keywords>                             
+                    <term key="Religion"/>
+                    <term key="ChristianLiterature"/>
+                    <term key="Magic"/>                                  
+                    <term key="Prayers"/>
+                    <term key="ZaMa"/>
+                </keywords>
+            </textClass>
+            <langUsage><language ident="am">Amharic</language>
```
Comment (eu-genia) [inline ParisBNF/abb/BNFabb162.xml:190]: English

File: ParisBNF/abb/BNFabb131.xml [hunk 9]
```diff
@@ -0,0 +1,194 @@
...
+                <keywords>
+                    
+                    <term key="ChristianLiterature"/>
+                    <term key="Theology"/>
+                    <term key="Translation"/>                
+                    <term key="ZaMa"/>
+                </keywords>
+            </textClass>
+            <langUsage><language ident="am">Amharic</language>
+            <language ident="gez">Gǝʿǝz</language></langUsage>
```
Comment (eu-genia) [inline ParisBNF/abb/BNFabb131.xml:181]: English

**Review by eu-genia — CHANGES_REQUESTED** [review]

File: ParisBNF/et/BNFet435.xml (hunk 2 above)
Comment (DenisNosnitsin1970) [reply ParisBNF/et/BNFet435.xml:29]: ok, changed.

File: ParisBNF/abb/BNFabb131.xml (hunk 1 above)
Comment (DenisNosnitsin1970) [reply ParisBNF/abb/BNFabb131.xml]: I wonder if I understood correctly.

File: ParisBNF/et/BNFet435.xml (hunk 3 above)
Comment (DenisNosnitsin1970) [reply ParisBNF/et/BNFet435.xml:36]: ok

File: ParisBNF/et/BNFet435.xml (hunk 4 above)
Comment (DenisNosnitsin1970) [reply ParisBNF/et/BNFet435.xml:92]: corrected

File: ParisBNF/et/BNFet348.xml (hunk 5 above)
Comment (DenisNosnitsin1970) [reply ParisBNF/et/BNFet348.xml:84]: corrected

File: ParisBNF/et/BNFet435.xml (hunk 6 above)
Comment (DenisNosnitsin1970) [reply ParisBNF/et/BNFet435.xml:88]: ok

File: ParisBNF/et/BNFet348.xml (hunk 7 above)
Comment (DenisNosnitsin1970) [reply ParisBNF/et/BNFet348.xml:80]: ok

File: ParisBNF/abb/BNFabb162.xml (hunk 8 above)
Comment (DenisNosnitsin1970) [reply ParisBNF/abb/BNFabb162.xml:190]: ok

File: ParisBNF/abb/BNFabb131.xml (hunk 9 above)
Comment (DenisNosnitsin1970) [reply ParisBNF/abb/BNFabb131.xml:182]: ok

**Review by CarstenHoffmannMarburg — APPROVED** [review]

**Review by CarstenHoffmannMarburg — APPROVED** [review]

File: ParisBNF/et/BNFet435.xml [hunk 10]
```diff
@@ -0,0 +1,102 @@
+<?xml version="1.0" encoding="UTF-8"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="BNFet435" xml:lang="en" type="mss">
+    <teiHeader xml:base="https://betamasaheft.eu/">
+        <fileDesc>
+            <titleStmt>
+                <title>Protective prayers</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="DR"/>
```
Comment (eu-genia) [inline ParisBNF/et/BNFet435.xml:9]: DN? please check everywhere

**Review by eu-genia — CHANGES_REQUESTED** [review]

File: ParisBNF/et/BNFet435.xml (hunk 10 above)
Comment (DenisNosnitsin1970) [reply ParisBNF/et/BNFet435.xml]: Sorry, corrected.

**Review by eu-genia — APPROVED** [review]

**Pull request merged**